<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabUi</class>
 <widget class="QWidget" name="TabUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QPushButton" name="homepageButton">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: rgb(125,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(100,100,100), stop: 1 rgb(100,100,100));
image: url(:/Resources/HomeIcon.png);
image-position: 1,1;
font-size: 18px;
}
QPushButton:hover{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(160,160,160), stop: 1 rgb(120,120,120));

}
QPushButton:pressed{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(120,120,120), stop: 1 rgb(80,80,80));
}</string>
     </property>
     <property name="text">
      <string>      HomePage</string>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="batteryButton">
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: rgb(125,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(100,100,100), stop: 1 rgb(100,100,100));
image: url(:/Resources/BatteryIcon.png);
image-position: 1,1;
font-size: 18px;
}
QPushButton:hover{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(160,160,160), stop: 1 rgb(120,120,120));
}
QPushButton:pressed{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(120,120,120), stop: 1 rgb(80,80,80));
}</string>
     </property>
     <property name="text">
      <string>      Battery</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="controlButton">
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: rgb(125,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(100,100,100), stop: 1 rgb(100,100,100));
image: url(:/Resources/SteeringIcon.png);
image-position: 1,1;
font-size: 18px;
}
QPushButton:hover{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(160,160,160), stop: 1 rgb(120,120,120));
}
QPushButton:pressed{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(120,120,120), stop: 1 rgb(80,80,80));
}</string>
     </property>
     <property name="text">
      <string>       Control</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="motorFaultButton">
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: rgb(125,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(100,100,100), stop: 1 rgb(100,100,100));
image: url(:/Resources/GearIcon.png);
image-position: 1,1;
font-size:18px;
}
QPushButton:hover{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(160,160,160), stop: 1 rgb(120,120,120));
}
QPushButton:pressed{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(120,120,120), stop: 1 rgb(80,80,80));
}</string>
     </property>
     <property name="text">
      <string>     MotorFault</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="motorButton">
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: rgb(125,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(100,100,100), stop: 1 rgb(100,100,100));
image: url(:/Resources/EngineIcon.png);
image-position: 1,1;
font-size: 18px;
}
QPushButton:hover{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(160,160,160), stop: 1 rgb(120,120,120));
}
QPushButton:pressed{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(120,120,120), stop: 1 rgb(80,80,80));
}</string>
     </property>
     <property name="text">
      <string>     Motor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="mpptButton">
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: rgb(125,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(100,100,100), stop: 1 rgb(100,100,100));
image: url(:/Resources/MPPTIcon.png);
image-position: 1,1;
font-size: 18px;
}
QPushButton:hover{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(160,160,160), stop: 1 rgb(120,120,120));
}
QPushButton:pressed{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(120,120,120), stop: 1 rgb(80,80,80));
}</string>
     </property>
     <property name="text">
      <string>         MPPT</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
